src/System.Private.CoreLib/shared/System/Text/UnicodeUtility.cs

```diff
@@ -11,7 +11,7 @@ internal static class UnicodeUtility
         /// <summary>
         /// The Unicode replacement character U+FFFD.
         /// </summary>
-        public const uint ReplacementChar = 0xFFFDU;
+        public const uint ReplacementChar = 0xFFFDu;
```

Comment:
Isn't there some strange corner case where the C# compiler might ignore the declared type of the `const` field and use the literal value instead, which could end up inadvertently making this an `int` or similar depending on the call site? I may be remembering the language rules incorrectly.

Reply:
That would sound like a compiler bug to me....

Reply:
I've asked some compiler folks to respond when they have a sec. It's likely I'm just misremembering or being overly paranoid.

Reply:
The type of the literal is technically an int, but it has a constant value that is in the range of `uint`, so there is an implicit constant expression conversion to `uint`. If the conversion were unsafe it would be an error.